src/options/modules/CustomOptionTriggers.js

```diff
@@ -265,11 +265,11 @@ function applyEmojiSearch(optionValue, option, event = {}) {
     if (optionValue.enabled) {
         document.getElementById("searchCopyAction").disabled = false;
         document.getElementById("emojipediaAction").disabled = false;
-        document.getElementById("searchBarDemo").removeAttribute("disabled");
```

Comment:
Why does not that work in Thunderbird?

Reply:
Oh, this was just a minor optimization I noticed while making the below changes to disable the Omnibox options.

Reply:
In what kind was this optimized? I see no preference in that way here compared to before?

Reply:
Presumably, setting an attribute directly is going to be faster that calling a function to do so. It it also consistent with the above lines.

Reply:
Well speed is not a good argument as there is no need to optimize, especially such micro-optimisations. _Especially_ if we don't even know whether there is even a performance gain here.

But okay, consistency makes sense I guess indeed… :upside_down_face: